ClientTests/SearchEngineTests.swift

```diff
@@ -58,12 +58,13 @@ class SearchEngineTests: XCTestCase {
         XCTAssertEqual(normalSearchURL, testSearchURL)
     }
 
+    /* This test is failing intermittently
     func testGetSuggestions() {
         client.getSuggestions(NORMAL_SEARCH, callback: { response, error in
             XCTAssertThrowsError(error)
```

Comment:
This is the line causing the intermittent errors in Unit Tests when running them in parallel... The test passed but the response is error is not thrown until later, and if it is thrown it happens at very different times make the test running in that moment to fail. I'm disabling it for now so that we can run tests in parallel and investigate it later (I will open an issue for that)

Reply:
Yeah, it may be related... I will open an issue to track that